backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.cloud.notifications.processors.email.aggregators;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+
+public class RhosakEmailAggregator extends AbstractEmailPayloadAggregator {
+    @Override
+    void processEmailAggregation(EmailAggregation aggregation) {
+        // TODO after the payloads are determined, notably the scheduled upgrade payload
```

Comment:
@gwenneg @josejulio I see that CI is complaining because of a constraint violation of payload as it cannot be `null`. I suspect it is coming from here. For now we do not have a template defined yet but that's in coming work so for now we are only looking to put the infrastructural code in. Is there a way to trick it so that the payload is not null?

Reply:
I was seeing the following exception in the logs
```
Couldn't persist aggregation!: javax.validation.ConstraintViolationException: Validation failed for classes [com.redhat.cloud.notifications.models.EmailAggregation] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='must not be null', propertyPath=accountId, rootBeanClass=class com.redhat.cloud.notifications.models.EmailAggregation, messageTemplate='{javax.validation.constraints.NotNull.message}'}
]
```

Wondering if this is something known or it is a result of this change.

Reply:
Answered here https://github.com/RedHatInsights/notifications-backend/pull/576#discussion_r705210051